src/include/nanobench.h

```diff
@@ -2991,6 +3014,16 @@ double Result::maximum(Measure m) const noexcept {
     return *std::max_element(data.begin(), data.end());
 }
 
+std::string const& Result::context(std::string const& variableName) const noexcept {
+    auto context = mConfig.mContext;
+    auto search = context.find(variableName);
+    static std::string NOTFOUND;
+    if (search == context.end()) {
+        return NOTFOUND;
```

Comment:
I think its better here to throw an exception when not found. That makes it obvious when e.g. a typo happens or somehthing is not provided. Otherwise one might wonder why the output doesn't have the data it should have.

Reply:
I agree. I just thought that this function should be `noexcept` given that all the other accessors are.